packages/cli/src/report/report-disk-writer.spec.ts

```diff
@@ -49,22 +50,22 @@ describe('ReportDiskWriter', () => {
         const format = 'json';
         const content = 'content';
 
-        const reportFileName = `${directory}/${fileName}.${format}`;
-
+        const reportFileName = `${fileName}.${format}`;
```

Comment:
this should be like report-disk.writer.ts, something like:

`import * as filenamify from 'filenamify';
import * as filenamifyUrl from 'filenamify-url';`
`${filenamify(fileName, { replacement: '_' })}.${format}`;
otherwise the test would fail if there's any special charterer

Reply:
i did not make any changes to your test. just made changes to return the file name instead of file path & validate the file name is being returned. In this test, you dont have any special character anyway.

Reply:
I know this is outta scope and it was missed in another review, but it's a tiny change and I commented about since you touched it here, anyway it could be done in another PR